corehq/apps/app_manager/models.py

```diff
@@ -4163,7 +4163,7 @@ def create_all_files(self):
         }
         tree = _parse_xml(files['profile.xml'])
 
-        def add_file_from_path(path, strict=False):
+        def add_file_from_path(path, strict=False, map=None):
```

Comment:
minor, but are there any other good names for this? i tend to lean against  overwriting built-in functions. when someone sees something like `if map:` it seems wrong since `map` would always be defined normally